apps/cookbook/src/app/showcase/showcase.routes.ts

```diff
@@ -151,6 +152,7 @@ export const routes: Routes = [
         path: 'chart',
         component: ChartShowcaseComponent,
       },
+      { path: 'stock-chart', component: StockChartShowcaseComponent },
```

Comment:
Consider something like `chart-stock` so it shows up besides the chart page.

Reply:
Done